daily-builds.json

```diff
@@ -0,0 +1,8 @@
+{
+  "versions": [
+    "v1.7.0",
+    "v1.6.0",
+    "v1.6.1",
+    "v1.5.0",
+  ],
+  "platforms": "linux/arm64,linux/amd64"
```

Comment:
arm64 is supported from 1.7.0, is this controlled somewhere else? I couldn't find

Reply:
Yes, but it won't break the previous releases if we build them with arm64 support :)